server/__init__.py

```diff
@@ -43,7 +43,7 @@ def create_app():
     cache.init_app(app)
 
     # apply the blueprints to the app
-    from routes import browser, factcheck, redirects, placelist, static, tools
+    from routes import browser, factcheck, redirects, placelist, static, tools, dev
```

Comment:
Please keep alphabetical ordering here and in the register_blueprints section

Reply:
Done